data/paradise world jobs.txt

```diff
@@ -160,13 +163,15 @@ mission "Transport partiers [2]"
 	cargo "snacks and school supplies" 2 4
 	to offer
 		random < 10
+	on offer
```

Comment:
Students probably don't care about the quality of their accommodations. Their party money is better off spent on... :beers: or newfangled space drugs. Even if the parents are bankrolling, that's just more money they could spend on party supplies.

Reply:
These are Paradise Planet students, though! I figure they're used to luxurious accommodations (I mean heck, their mothers are sending them a literal ton of brownie mix in school), and it also becomes a statement about the culture of the Paradise Planets.